orchestrator/dockerdir/usr/bin/add_mysql_nodes.sh

```diff
@@ -29,30 +28,49 @@ am_i_leader() {
         return 1
     fi
 
+    echo '[INFO] I am a leader'
     return 0
 }
 
 discover() {
     local host=$1
     local port=$2
 
-    curl "${ORC_HOST}/api/discover/${host}/${port}"
+    HOSTNAME=$(curl -s "${ORC_HOST}/api/instance/${host}/${port}" | jq '.InstanceAlias' | tr -d 'null')
+    if [ -n "$HOSTNAME" ]; then
+        echo "[INFO] The Mysql node ${host} is already present in orchestrator. Skipping ..."
+        return 0
+    fi
```

Comment:
Orchestrator already perform this check, should we check it also? 

https://github.com/openark/orchestrator/blob/master/go/logic/orchestrator.go#L228-L234

Reply:
Yes, but I want to see clear log and user should understand what is going on.